scripts/ci-make.sh

```diff
@@ -19,5 +19,4 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
-
-cd $REPO_ROOT && make images
+cd "${REPO_ROOT}" && make images
```

Comment:
It might be a good idea to update this to run make all, similar to the main cluster-api script here.